src/heliaServer.ts

```diff
@@ -85,7 +85,13 @@ export class HeliaServer {
         path: '/api/v0/repo/gc',
         type: 'POST',
         handler: async (request, reply): Promise<void> => this.gc({ request, reply })
-      }, {
+      },
+      {
+        path: '/api/v0/repo/gc',
+        type: 'GET',
+        handler: async (request, reply): Promise<void> => this.gc({ request, reply })
+      },
```

Comment:
triggering GC when debugging made not having GET here annoying. it was falling through to our `/` path and trying to call `.fetch`